README.md

```diff
@@ -92,9 +92,11 @@ moment.loadPersian([options])
 
 ### Options
 
-| Param            | Type    | Default | Description                               | Example                                      |
-| ---------------- | ------- | ------- | ----------------------------------------  | -------------------------------------------- |
+| Param            | Type    | Default   | Description                               | Example                                      |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
 | usePersianDigits | Boolean | `false`   | Use persian digits (Use at your own risk) | `moment.loadPersian({usePersianDigits: true})` |
+| ---------------- | ------- | -------   | ----------------------------------------  | -------------------------------------------- |
+| dialect          | String  | `persian` | Available values = `persian`, `persian-modern` | `moment.loadPersian({dialect: 'persian-modern'})` |
```

Comment:
Please change to `dialect *` to indicate a **note** and add a description below the table describing what this option changes.